src/PKSim.Infrastructure/Services/SimulationExportTask.cs

```diff
@@ -79,6 +79,18 @@ public Task ExportResultsToCSVAsync(Simulation simulation)
             fileName => ExportResultsToCSVAsync(simulation, fileName), Constants.DirectoryKey.REPORT);
       }
 
+      public Task ExportSimulationToCppAsync(Simulation simulation)
+      {
+         var outputFolder = _dialogCreator.AskForFolder(PKSimConstants.UI.ExportSimulationToCpp, Constants.DirectoryKey.SIM_MODEL_XML);
+         return string.IsNullOrEmpty(outputFolder) ? Task.CompletedTask : ExportSimulationToCppAsync(simulation, outputFolder);
+      }
+
+      public Task ExportSimulationToCppAsync(Simulation simulation, string outputFolder)
+      {
+         _lazyLoadTask.Load(simulation);
+         return Task.Run(() => _simModelExporter.ExportCppCode(_coreSimulationMapper.MapFrom(simulation, shouldCloneModel: false), outputFolder, FormulaExportMode.Formula));
```

Comment:
in crashes in core with the example, in PKsim, it does nothing.... I have no idea what's going on :)

Reply:
@msevestre where is the test which crashes when exporting to c++?

Reply:
In core. It's commented out

Reply:
https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/blob/develop/tests/OSPSuite.Core.IntegrationTests/SimModelExporterSpecs.cs#L159

Reply:
in pksim, it does not do anything. no crash, no files nothing